rpc_jobs/standard_job_postmerge.yml

```diff
@@ -0,0 +1,141 @@
+        common.shared_slave() {{
+          pubcloud.runonpubcloud {{
+            // Set the default environment variables used
+            // by the artifact and test result collection.
+            env.RE_HOOK_ARTIFACT_DIR="${{env.WORKSPACE}}/artifacts"
+            env.RE_HOOK_RESULT_DIR="${{env.WORKSPACE}}/results"
+
+            // Set the job result default
+            currentBuild.result="SUCCESS"
+
+            try {{
+              ansiColor('xterm') {{
+                dir("${{env.WORKSPACE}}/${{env.ghprbGhRepository}}") {{
+                  withCredentials(common.get_cloud_creds()) {{
+
+                    stage('Checkout') {{
+                      git branch: env.BRANCH, url: env.REPO_URL
+                    }} // stage
+
+                    stage('Execute Pre Script') {{
+                      // Retry the 'pre' stage 3 times. The 'post' stage is considered
+                      // to be postparation for the test, so let's try and make sure
+                      // it has the best chance of success possible.
+                      retry(3) {{
+                        sh """#!/bin/bash -xeu
+                          if [[ -e gating/post_merge_test/pre ]]; then
+                            gating/post_merge_test/pre
+                          fi
+                        """
+                      }}
+                    }} // stage
+
+                    try{{
+                      stage('Execute Run Script') {{
+                        sh """#!/bin/bash -xeu
+                          gating/post_merge_test/run
+                        """
+                      }} // stage
+                    }} finally {{
+                      stage('Execute Post Script') {{
+                        // We do not want the 'post' execution to fail the test,
+                        // but we do want to know if it fails so we make it only
+                        // return status.
+                        post_result = sh(
+                          returnStatus: true,
+                          script: """#!/bin/bash -xeu
+                                     if [[ -e gating/post_merge_test/post ]]; then
+                                       gating/post_merge_test/post
+                                     fi"""
+                        )
+                        if (post_result != 0) {{
+                          print("Post-Merge Test (post) failed with return code ${{post_result}}")
+                        }} // if
+                      }} // inner try
+                    }} // stage
+                  }} // withCredentials
+                }} // dir
+              }} // ansiColor
+            }} finally {{
+              stage("Collect Test Results"){{
+                dir(env.RE_HOOK_RESULT_DIR){{
+                  junit allowEmptyResults: true, testResults: '*.xml'
+                }}
+              }}
+              stage("Archive Artifacts"){{
+                common.archive_artifacts()
+              }}
```

Comment:
It doesn't appear failure is logged in a ticket, how will these jobs be monitored?

Reply:
Added ticket creation